http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -416,4 +420,29 @@ private static boolean hasScope(String scopeParam, String targetScope) {
         }
         return false;
     }
+
+    private String addDeploymentAndQueryScopes(String paramValue){
+        String combinedScopes = null;
+        Set<String> allScopes = new HashSet<String>();
+        if (paramValue != null && !paramValue.isEmpty()) {
+            Set<String> queryScopes = new HashSet<String>(Arrays.asList(paramValue.split("\\s+")));
+            allScopes.addAll(queryScopes);
```

Comment:
I think we could just use `Arrays.asList...` here, we don't need the `queryScopes` variable.